Examples/Shared/R4/resources/ValueSet-R4-immobilization.json

```diff
@@ -0,0 +1,64 @@
+{
+    "resourceType": "ValueSet",
+    "id": "2.16.840.1.113883.17.4077.3.2006",
+    "meta": {
+      "versionId": "1",
+      "lastUpdated": "2020-12-10T17:27:10.934+05:30",
+      "source": "#dLfN1XZj92k3Xpvr"
+    },
+    "identifier": [
+      {
+        "system": "urn:ietf:rfc:3986",
+        "value": "2.16.840.1.113883.17.4077.3.2006"
```

Comment:
This is in VSAC.

https://vsac.nlm.nih.gov/valueset/2.16.840.1.113883.17.4077.3.2006/expansion/Latest